docusaurus/docs/Android/03-ui/06-guides/02-building-message-list-screen.mdx

```diff
@@ -150,9 +162,9 @@ messageListView.setMessageEditHandler(message ->
 );
 
 // Handle navigate up state
-messageListViewModel.getState().observe(this, state -> {
+messageListViewModel.getState().observe(getViewLifecycleOwner(), state -> {
     if (state instanceof MessageListViewModel.State.NavigateUp) {
-        // Handle navigate up
+        requireActivity().finish();
```

Comment:
I think this is a part of the ktx libraries.
Not sure but don't think this is available in java.